runtime/src/bank.rs

```diff
@@ -1972,6 +1972,10 @@ impl Bank {
         );
         bank.fill_missing_sysvar_cache_entries();
 
+        if !bank.bank_hash_skips_rent_rewrites() {
```

Comment:
I kind of think the 'if' belongs inside `rebuild_skipped_rewrites`. This way we can activate the feature and verify it does nothing. Otherwise, the 'if' is inside a more complicated fn that may be harder to test? I didn't look at the details enough to know that the feature activation aspect of this is being tested. Again, we'll know as soon as the feature is activated and the next time a validator is restarted. So, I'm not worried. I'm just generally worried about feature activation testing.